_posts/2024-11-27-driving-community-contributions.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: post
+title: "Give back and go forward: Driving community contributions from vendor led to vendor neutral"
+authors:
+   - anandhi
+   - jamesmcintyre
+date: 2024-11-27
+categories: 
+    - community
+meta_keywords: 
+meta_description: 
```

Comment:
_meta_keywords:_ OpenSearch Project, AWS open source, Linux Foundation, community contributions, OpenSearch Software Foundation

_meta_description:_ Explore the OpenSearch journey from an AWS-led project to a vendor-neutral, community-driven open-source platform under the Linux Foundation. Learn about the project's growth, challenges, and future in search, analytics, and vector databases.